pep-0696.rst

```diff
@@ -434,6 +434,10 @@ using the "=" operator inside of the square brackets like so:
    type Qux[*Ts = *tuple[str]] = Ham[*Ts]
    type Rab[U, T = str] = Bar[T, U]
 
+Similarly to the bound for a type parameter, defaults should be
```

Comment:
We should cross-reference https://peps.python.org/pep-0695/#scoping-behavior here and say that the scoping behavior of defaults is the same as the "def695" scopes I introduced there.